packages/fields/src/actions/base-post/index.ts

```diff
@@ -0,0 +1,5 @@
+export { default as viewPost } from './view-post';
```

Comment:
Do we need the "base-post" folder?

Reply:
I don't have a strong opinion, but I explained why I structured it in this way in the PR description! Happy to revisit it!

Reply:
I like to start flat personally in general but I don't mind if you prefer it this way.